prody/proteins/interactions.py

```diff
@@ -2124,6 +2193,10 @@ def calcStatisticsInteractions(data, **kwargs):
         else: pass
     
     statistic.sort(key=lambda x: x[1], reverse=True)
+    
+    if statistic == []:
+        LOGGER.info("Decrease the value of weight_cutoff parameter to obtain the results. Default is 0.2.")
```

Comment:
This could be a bit clearer. Perhaps say that results were filtered out or whatever it is that's happening.

Reply:
Makes sense but too long for message. How about "No meaningful interactions found. Decrease weight_cutoff to obtain some results (default is 0.2)."?

Reply:
I replaced it.

Reply:
Looks good. Ready to merge